.meta/sources/docker.toml

```diff
@@ -0,0 +1,33 @@
+[sources.docker]
+beta = true
+delivery_guarantee = "best_effort"
+guides = []
+output_types = ["log"]
+resources = []
+through_description = "the docker engine daemon"
+
+[sources.docker.options.include_containers]
+type = "[string]"
+examples = [
+    "<container-id>",
+    "ffd2bc2cb74a"
+]
+null = true
+description = """\
+A list of container ids to match against when filtering running containers. This \
+will attempt to match the container id from the beginning meaning you do not need \
+to include the whole id but just the first few characters. If no containers ids are \
+provided, all containers will be included.\
+"""
+
+[sources.docker.options.include_labels]
+type = "[string]"
+examples = [
+    "latest",
```

Comment:
Are these realistic examples that someone would use when setting up the docker source? Just double-checking.